tests/api/testdata/generators.py

```diff
@@ -87,7 +87,7 @@ def _fill_pytest_param(
         marks.append(allure.link(url=endpoint.spec_link, name="Endpoint spec"))
     param_id = f"{endpoint.path.replace('/', '_')}_{method.name}_{positive_str}"
     if sum(1 for e in Endpoints if e.path == endpoint.path) > 1:
-        param_id += f"_{endpoint.endpoint.get_data_class_pretty_name()}"
+        param_id += f"_{endpoint.pretty_name}"
```

Comment:
two lines above can be in pretty_name realization too

Reply:
discussed, reworked